addon/templates/components/frost-file-picker.hbs

```diff
@@ -3,10 +3,13 @@
 </div>
 <div class="background">
   {{frost-text
+    hook=hook
+    isHookEmbedded=true
```

Comment:
It's a property provided by frost text so that if it is embedded inside another component the hook name doesn't get duplicated.